Sources/SwiftPlot/Renderer.swift

```diff
@@ -146,15 +146,15 @@ public protocol Renderer: AnyObject{
                            fillColor: Color)
 
     /*drawSolidPolygon()
-    *params: points: [Point],
-    *                 fillColor: Color,
-    *                 isOriginShifted: Bool
-    *description: Draws a polygon with specified fill color from an array of Points
+    *params: polygon: Polygon,
+    *        fillColor: Color,
+    *        isOriginShifted: Bool
+    *description: Draws a polygon with specified fill color from a Polygon struct
     *             This function can operate in both coordinate systems with and
     *             without shifted origin.
     *             This is decided by the boolean parameter isOriginShifted.
     */
-    func drawSolidPolygon(points: [Point],
+    func drawSolidPolygon(polygon: Polygon,
```

Comment:
We can remove the first argument label; there's no need to say `polygon` again since it's already in the base name and parameter's type name.